AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Task/AWSAuthSignOutTask.swift

```diff
@@ -62,9 +62,18 @@ class AWSAuthSignOutTask: AuthSignOutTask, DefaultLogger {
     }
 
     private func sendSignOutEvent() async {
+
+        let presentationAnchor: AuthUIPresentationAnchor?
```

Comment:
Not sure how this would work on watchOS and tvOS, the` typealias` is already wrapped around iOS and macOS..

Reply:
ahh.. nvm.. I see it in `AuthUIPresentationAnchorPlaceholder `